source/internal/_clone.js

```diff
@@ -30,7 +30,8 @@ export default function _clone(value, deep, map) {
     map.set(value, copiedValue);
 
     for (var key in value) {
-      if (value.hasOwnProperty(key)) {
+      // If there is no prototype, the property must be its own
+      if (!_hasPrototype(value) || value.hasOwnProperty(key)) {
```

Comment:
This is not precisely an objection.  I'm curious why you chose to implement it this way rather than as in this (untested) snippet:

```js
var hasOwnProperty = Object .prototype .hasOwnProperty;
/* ... */

for (var key in value) {
    if (hasOwnProperty .call (value, key) {
/* ... */
```

Reply:
@CrossEye When this patch was created, I was a bit concerned about the readability. Hence the comment as well. I am all up for doing this the concise way. Let me update this.